ArjunaCore/arjuna/classes/com/arjuna/ats/internal/arjuna/recovery/PeriodicRecovery.java

```diff
@@ -719,6 +770,12 @@ private void doWorkInternal()
     {
         // n.b. we only get here if status is SCANNING
 
+        /* Let's assume that we want to block the suspension of the Recovery Manager.
+         * Recovery Modules implementing SuspendBlockingRecoveryModule should prove
+         * this assumption wrong :-)
```

Comment:
See my earlier comment about nothing clearing this flag if there are no configured instances of SuspendBlockingRecoveryModule.

Reply:
Done